components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -263,6 +263,13 @@ public enum ErrorMessages {
         ERROR_CODE_UNAUTHORIZED_FRAGMENT_APP_ACCESS("60066", "Fragment applications can be managed only " +
                 "from the organization that fragment application resides.", "Application %s can't be managed " +
                 "from organization %s."),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_EMPTY_REQUEST_BODY("60067", "Invalid request.",
+                "At least one of the attributes from shareWithAllChildren and sharedOrganizations should be present."),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_REQUEST_BODY("60068", "Invalid request.", "Cannot share " +
+                "the application with a set of child organizations when shareWithAllChildren is set."),
+        ERROR_CODE_INVALID_DELETE_SHARE_REQUEST("60069", "Invalid request.", "Cannot delete " +
+                "sharing the application of id: %s to organization of id: %s when it is shared with all child " +
```

Comment:
Shall we refactor the description here

suggestions:
1. If the app id is the fragment app id and org id is the fragment app owns org: "Cannot delete the fragment application with ID %s from the organization with ID %s if the main application is shared with all children organizations. "
2. If the app id is main app id and org id is fragment app belonging org: "Cannot unshare the application with ID %s from the organization with ID %s if the application is shared with all children organizations."

Reply:
Fixed with c69ad158b6ad258092ef3dc81638714283a84e98